lib/parsers.ml

```diff
@@ -10,7 +10,7 @@ let is_tab = function
   | _ -> false
 
 let is_tab_or_space = function
-  | '\t' | ' ' | '\026' | '\012' | '\009'   -> true
+  | '\t' | ' ' | '\026' | '\012' -> true
```

Comment:
What's the reason to remove `\009`? It's the horizontal tab, right?

Reply:
\009 = '\t' 
![image](https://user-images.githubusercontent.com/5608710/109089776-c1f1c800-774c-11eb-910f-5b7137987079.png)

Reply:
ah, got it